<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_3D_TESTClass</class>
 <widget class="QMainWindow" name="QT_3D_TESTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT_3D_TEST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>821</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="left_plus_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Left +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_minus_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Left -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_minus_">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Right -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_plus_">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Right +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bottom_plus_">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bottom +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bottom_minus_">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bottom +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="top_minus_">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Top -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="top_plus_">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Top -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="znear_plus_">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>zNear +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="znear_minus_">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>zNear -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zfar_plus_">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>zFar +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zfar_minus_">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>zFar -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_plus_">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_plus_">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_plus_">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>840</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Z +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_minus_">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Z -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_minus_">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_minus_">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X -</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_3d_test.qrc"/>
 </resources>
 <connections/>
</ui>
